rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -402,7 +402,7 @@ public static boolean isValidPegInTx(
                         throw new ScriptException(message);
                     }
                     Script inputStandardRedeemScript = redeemScriptParser.extractStandardRedeemScript();
-                    if (activeFederations.stream().anyMatch(federation -> federation.getStandardRedeemScript().equals(inputStandardRedeemScript))) {
+                    if (activeFederations.stream().anyMatch(federation -> (federation instanceof ErpFederation ? ((ErpFederation) federation).getStandardRedeemScript() : federation.getRedeemScript()).equals(inputStandardRedeemScript))) {
```

Comment:
That's a long line, perhaps we can reduce it
```suggestion
if (activeFederations.stream().anyMatch(federation -> {
                        Script fedStandardRedeemScript = federation instanceof ErpFederation ?
                            ((ErpFederation) federation).getStandardRedeemScript() :
                            federation.getRedeemScript();
                         return fedStandardRedeemScript.equals(inputStandardRedeemScript);
                    })) {
```